data/weapons.txt

```diff
@@ -1089,6 +1089,7 @@ outfit "Gatling Gun"
 		"burst reload" 1
 		"firing energy" .03
 		"firing heat" 2.5
+		"firing force" 5
```

Comment:
personally, i think this change would improve the lifespan of this weapon on smaller escort ships, but i'm not super hung-up over it...  in the end, the slowdown would only last long enough for the weapon to cool-down, at which time, the tiny ship takes off again, and resumes normal behaviour.

this will probably pop up again as a separate pr...